<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DICTDialogConfig</class>
 <widget class="QDialog" name="DICTDialogConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>848</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DICT</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="button_box">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Informations</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_15">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Coordonnées</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0" colspan="3">
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Dénomination</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="coordDenom"/>
            </item>
           </layout>
          </item>
          <item row="1" column="0" colspan="3">
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Personne</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="coordPersonne"/>
            </item>
           </layout>
          </item>
          <item row="2" column="0" colspan="3">
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Numéro et voie</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="coordNumVoie"/>
            </item>
           </layout>
          </item>
          <item row="3" column="0" colspan="3">
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Lieu Dit /BP</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="coordBP"/>
            </item>
           </layout>
          </item>
          <item row="4" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Code Postal</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="coordCP"/>
            </item>
           </layout>
          </item>
          <item row="4" column="1" colspan="2">
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Commune</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="coordCommune"/>
            </item>
           </layout>
          </item>
          <item row="5" column="0" colspan="2">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Téléphone</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="coordTel"/>
            </item>
           </layout>
          </item>
          <item row="5" column="2">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Fax</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="coordFax"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Responsable du dossier</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_18">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Nom</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="respNom"/>
            </item>
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Service</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="respService"/>
            </item>
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Téléphone</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="respTel"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Signataire</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_19">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Nom</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="signNom"/>
            </item>
            <item>
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Signature</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="signSignature"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Informations relatives à la dégradation des ouvrages</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_20">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_12">
            <item>
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>Numéro de téléphone</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="TelEndommagement">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>Numéro du SDIS</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="Endommagement"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Configuration</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_17">
       <item>
        <widget class="QGroupBox" name="groupBox_7">
         <property name="title">
          <string>Dossiers</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_13">
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>Répertoire d'importation des fichiers xml :</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLineEdit" name="configRepXML"/>
            </item>
            <item row="1" column="1">
             <widget class="QToolButton" name="toolButtonXML">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Répertoire de sortie du plan et du formulaire :</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLineEdit" name="configRep"/>
            </item>
            <item row="1" column="1">
             <widget class="QToolButton" name="toolButton">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Formatage des noms des fichiers en sortie</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_16">
          <item>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0" colspan="2">
             <widget class="QLabel" name="label_25">
              <property name="text">
               <string>Récépissé</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_27">
              <property name="text">
               <string>Préfixe :</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="prefRecep"/>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="label_28">
              <property name="text">
               <string>Suffixe :</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QLineEdit" name="sufRecep"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="label_29">
              <property name="text">
               <string>Plan</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_30">
              <property name="text">
               <string>Préfixe :</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="prefPlan"/>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="label_31">
              <property name="text">
               <string>Suffixe :</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QLineEdit" name="sufPlan"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_11">
            <item>
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>Extension du numéro de dossier</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="configExtension"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_8">
         <property name="title">
          <string>En cas de DT</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_23">
          <item>
           <widget class="QCheckBox" name="casDT">
            <property name="text">
             <string>Souhaitez vous automatiquement cocher la case
 &quot;(cas d'un récépissé de DT) si le récépissé est pour une DT ?</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>Choix du moteur de rendu du formulaire cerfa</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_21">
          <item>
           <widget class="QRadioButton" name="radioQGisComposer">
            <property name="text">
             <string>Composeur QGis</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioPoppler">
            <property name="text">
             <string>Poppler (nécessite l'installation de pypoppler-Qt5 et de ses dépendances)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_9">
         <property name="title">
          <string>Fusion des PDF</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_22">
          <item>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QLabel" name="label_26">
              <property name="text">
               <string>Veuillez sélectionner où se trouve le logiciel pdftk :</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLineEdit" name="configPDFTK"/>
            </item>
            <item row="1" column="1">
             <widget class="QToolButton" name="toolButtonPDFTK">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="fusionPDF">
            <property name="text">
             <string>Souhaitez-vous fusionner les documents (plans et récépissé) pour l'envoi ?</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_10">
         <property name="title">
          <string>Rappel</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <widget class="QLabel" name="label_19">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>491</width>
            <height>61</height>
           </rect>
          </property>
          <property name="text">
           <string>Afin d'assurer le fonctionnement de l'impression il est important de créer dans la composition 
de votre choix une couche carte avec pour ID : &quot;carte_1&quot;</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>coordDenom</tabstop>
  <tabstop>coordPersonne</tabstop>
  <tabstop>coordNumVoie</tabstop>
  <tabstop>coordBP</tabstop>
  <tabstop>coordCP</tabstop>
  <tabstop>coordCommune</tabstop>
  <tabstop>coordTel</tabstop>
  <tabstop>coordFax</tabstop>
  <tabstop>respNom</tabstop>
  <tabstop>respService</tabstop>
  <tabstop>respTel</tabstop>
  <tabstop>signNom</tabstop>
  <tabstop>signSignature</tabstop>
  <tabstop>TelEndommagement</tabstop>
  <tabstop>Endommagement</tabstop>
  <tabstop>configRep</tabstop>
  <tabstop>toolButton</tabstop>
  <tabstop>configRepXML</tabstop>
  <tabstop>toolButtonXML</tabstop>
  <tabstop>configExtension</tabstop>
  <tabstop>casDT</tabstop>
  <tabstop>button_box</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>DICTDialogConfig</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>DICTDialogConfig</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
